scripts/devtech/devtech_machines.zs

```diff
@@ -29,10 +32,535 @@ var stock_manipulator = IRecipeMap.recipeMapBuilder("stock_manipulator")
+    .outputs(<gregtech:ore_pentlandite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(24))
+    .inputs(<contenttweaker:coin_steel> * 3)
+    .EUt(30)
+    .duration(30)
+    .outputs(<gregtech:ore_tenorite_0> * 32)
+    .outputs(<gregtech:ore_copper_0> * 32)
+    .outputs(<gregtech:ore_cuprite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(25))
+    .inputs(<contenttweaker:coin_steel> * 3)
+    .EUt(30)
+    .duration(30)
+    .outputs(<gregtech:ore_tetrahedrite_0> * 32)
+    .outputs(<gregtech:ore_copper_0> * 32)
+    .outputs(<gregtech:ore_stibnite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(26))
+    .inputs(<contenttweaker:coin_steel> * 5)
+    .EUt(30)
+    .duration(50)
+    .outputs(<gregtech:ore_tin_0> * 32)
+    .outputs(<gregtech:ore_cassiterite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(27))
+    .inputs(<contenttweaker:coin_steel> * 10)
+    .EUt(30)
+    .duration(100)
+    .outputs(<gregtech:ore_topaz_0> * 32)
+    .outputs(<gregtech:ore_blue_topaz_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(28))
+    .inputs(<contenttweaker:coin_steel> * 5)
+    .EUt(30)
+    .duration(50)
+    .outputs(<gregtech:ore_chalcopyrite_0> * 32)
+    .outputs(<gregtech:ore_iron_0> * 32)
+    .outputs(<gregtech:ore_pyrite_0> * 32)
+    .outputs(<gregtech:ore_copper_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(29))
+    .inputs(<contenttweaker:coin_steel> * 5)
+    .EUt(30)
+    .duration(50)
+    .outputs(<gregtech:ore_brown_limonite_0> * 32)
+    .outputs(<gregtech:ore_yellow_limonite_0> * 32)
+    .outputs(<gregtech:ore_banded_iron_0> * 32)
+    .outputs(<gregtech:ore_malachite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(30))
+    .inputs(<contenttweaker:coin_steel> * 5)
+    .EUt(30)
+    .duration(50)
+    .outputs(<gregtech:ore_magnetite_0> * 32)
+    .outputs(<gregtech:ore_iron_0> * 32)
+    .outputs(<gregtech:ore_vanadium_magnetite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(31))
+    .inputs(<contenttweaker:coin_steel> * 5)
+    .EUt(30)
+    .duration(50)
+    .outputs(<gregtech:ore_nether_quartz_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(32))
+    .inputs(<contenttweaker:coin_steel> * 5)
+    .EUt(30)
+    .duration(50)
+    .outputs(<gregtech:ore_garnierite_0> * 32)
+    .outputs(<gregtech:ore_nickel_0> * 32)
+    .outputs(<gregtech:ore_cobaltite_0> * 32)
+    .outputs(<gregtech:ore_pentlandite_0> * 32)
+    .buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+    .notConsumable(intCircuit(33))
```

Comment:
I think using @hjae78's suggestion of two config circuits will be the best solution for getting around the issue of being limited to 32 configurations. Every recipe will need two config circuits no matter what, however. Also note that recipes with a circuit combination of: `1, 2` and another of `2, 1` WILL conflict, so this will also need to be taken into consideration.

Reply:
kinda like dominoes? 1,1 -> 1,32 and then 2,2 -> 2,32?

Reply:
You can do `0, 0-32`, then `1, 1-32`, then `2, 2-32`, etc. Not quite `32^2` options, but it should definitely be enough.

Reply:
DONE POG